### DISCUSSION TITLE
Community Roles Application

### DISCUSSION THREAD

**Discussion by AlmasB** [AlmasB/FXGL#1027] [General]
If you want to be involved with the project on a long-term basis, please apply for a community role by replying to this post.

## Community Roles

Community developers may apply to any of these roles by opening an issue or discussing in the [community chat](https://gitter.im/AlmasB/FXGL). Each role is allocated based on discussions with the individual. The individual is expected to be **committed** to the role guidance. However, the role descriptions are merely guidelines and not binding in any way. The role may be revoked at the request of the individual or due to violation of the [Code of Conduct](CODE_OF_CONDUCT.md).

#### Maintainer (Collaborator):

* Lead the project development and set out project goals
* Represent the project at conferences and other events
* Engage with the community to obtain feedback
* Review Pull Requests
* Guide new and existing Contributors to develop Pull Requests
* Provide technical support in the community chat

#### Coordinator:

* Lead one or more areas of the project development
* Engage with the Maintainers to discuss the development
* Review Pull Requests
* Guide new and existing Contributors to develop Pull Requests
* Provide technical support in the community chat

#### Tester:

* Provide feedback on latest project versions
* Crash test new and existing features after major updates
* Provide technical support in the community chat

#### Contributor:

Any developer whose Pull Request has been merged is considered a Contributor.

#### Sponsor:

Any developer who is sponsoring (or has sponsored in the past) the project is considered a Sponsor.

**Comment by carldea** [comment]
@AlmasB I would like to be a tester role. If time permits I hope to contribute. Thanks! Keep up the great work!

**Reply by AlmasB** [reply]
Welcome aboard @carldea! Great to have you with us!

Most of the game demos are at: https://github.com/AlmasB/FXGLGames

```
cd to one of the games
mvn javafx:run
```

If something doesn't work and/or can be improved, give me or @adambocco a shout via issues.

The community chat is on [Gitter](https://gitter.im/AlmasB/FXGL). The Q&A is slowly transitioning to GitHub Discussions.

I'll update the README with your new role soon. Many thanks for your interest!

**Reply by AlmasB** [reply]
@carldea Done - [commit](https://github.com/AlmasB/FXGL/commit/e530d910bf374cca19823d094f98d668432666ed)
